Flake8 wrapper doesn't annotate all errors

First of all, great action! Not only does it work, but it also taught me about problem matchers. Super useful 🚀 

The action does create annotations as advertised; however, it seems to miss some of them for the flake8 linter. Flake8 finds a total of 16 issues, but only 10 of them are annotated. The following lines are missed by the matcher:

The code line `## An added line that breaks the linter` I've added specifically to test this produces

```
.\imageio\core\functions.py:620:1: E266 too many leading '#' for block comment
```

as expected, but is not annotated.

Some errors in files not touched by a PR are found in the action, but also not annotated (this is a beta feature on GH, so I guess it may be unrelated to this repo). Examples

```
.\imageio\core\format.py:638:13: E731 do not assign a lambda expression, use a def
.\imageio\core\util.py:19:1: E402 module level import not at top of file
.\imageio\core\util.py:186:1: E402 module level import not at top of file
.\imageio\core\util.py:226:9: E731 do not assign a lambda expression, use a def
```


Here is an example commit from the repo where I've set things up:
https://github.com/imageio/imageio/pull/614/commits/38b2d86654fa3b8cce879e21c9fe842d45f1881c

And here is the accompanying Action log: 
https://github.com/imageio/imageio/runs/2207580247?check_suite_focus=true

--- a/README.md
+++ b/README.md
@@ -94,3 +94,46 @@ For details, sources and licenses of individual problem matchers, see
 [problem-matchers/](problem-matchers/).
 
 Pull Requests with additional linters welcome!
+
+### Annotation Limitations
+
+GitHub imposes limitations on the total number of annotations a step or job may create [[Source 1](https://github.community/t/annotation-limitation/17998/2)][[Source 2](https://github.community/t/maximum-number-of-annotations-that-can-be-created-using-github-actions-logging-commands/16991)]. If a linter finds more problems than what may be annotated, only a random subset of the reported errors/warnings will be annotated. In either case, all found errors are reported in the action's logs. The exact limitations are:
+
+- 10 warning annotations and 10 error annotations per step
+- 50 annotations per job (sum of annotations from all the steps)
+- 50 annotations per run (separate from the job annotations, these annotations aren’t created by users
+
+
+One approach to (partially) counteract this limitation is to split the linting into multiple steps/jobs. This can be done on a per linter or per folder basis. Example:
+
+```yaml
+jobs:
+  # this job can annotate up to 50 errors/warnings
+  lint_core:
+    ...
+    steps:
+      ...
+      # this step can annotate up to 10 errors/warnings
+      - name: Flake8 Module1
+        uses: liskin/gh-problem-matcher-wrap@v1
+        with:
+          linters: flake8
+          run: flake8 src/core/module1
+      # this step can annotate an additional 10
+      - name: Flake8 Module2
+        uses: liskin/gh-problem-matcher-wrap@v1
+        with:
+          linters: flake8
+          run: flake8 src/core/module2
+  # this job can annotate an additioanal 50 errors/warnings
+  lint_plugins:
+    ...
+    steps:
+      ...
+      # this step can annotate up to 10 errors/warnings
+      - name: Flake8
+        uses: liskin/gh-problem-matcher-wrap@v1
+        with:
+          linters: flake8
+          run: flake8 src/plugins/ 
+```

